src/lex.rs

```diff
@@ -128,6 +128,11 @@ impl<'a> Lexer<'a> {
             error_handler: ErrorHandler::new(),
         }
     }
+    /// Return the current location of the lexer
+    pub fn location(&self) -> Location {
+        self.location
+    }
```

Comment:
Is this just for ergonomics?

Reply:
https://github.com/jyn514/rcc/pull/172/files#diff-b4aea3e418ccdb71239b96952d9cddb6R79

Reply:
The lexer location is private so people can't modify it. That would cause errors later when I implement spans because they could break the invariant that spans start before they end.